src/main/java/com/xiaomi/infra/pegasus/rpc/async/MetaSession.java

```diff
@@ -122,13 +142,36 @@ public Void call() throws Exception {
     }
   }
 
+  public final void executeWithTimeout(client_operator op, long timeoutMs) {
+    this.execute(op, 0, true, timeoutMs);
+  }
+
+  public final void asyncQuery(client_operator op, Runnable callbackFunc, int maxQueryCount) {
```

Comment:
What is used for?  you has change meta request to more general `asyncExecute`, whiy still contain it?

Reply:
yes, you're right~